modules/aggregation-layers/bundle.ts

```diff
@@ -1,3 +1,7 @@
 export * from '../layers/bundle/peer-dependency';
 
-export * from './src';
+// Import from package name instead of relative path
+// This will be resolved to src or dist by esbuild depending on bundle settings
+// dist has TS transformers applied
+/* eslint-disable import/no-extraneous-dependencies */
+export * from '@deck.gl/aggregation-layers';
```

Comment:
Why can't we keep `./src`? This seems confusing to me

Reply:
With this PR Babel is removed from the building process. Typescript is used to compile the ESM entry point and esbuild is used for the bundle. Esbuild does not use tsc to process ts files, but its own parser. So whatever old Babel plug-ins we have we will have to rewrite twice, one for tsc and one for esbuild, unless we force esbuild to bundle from the js files emitted by tsc.

By importing from the package name esbuild resolves to the ESM dist instead of src.